esp/services/ws_resources/ws_resourcesService.cpp

```diff
@@ -76,16 +77,20 @@ bool CWsResourcesEx::onQueueQuery(IEspContext& context, IEspQueueQueryRequest& r
             else
                 type = CHPCCQueueType_HThor;
 
+            SCMStringBuffer clusterName;
+            cluster.getName(clusterName);
+            if ((type == CHPCCQueueType_Roxie) && cluster.isQueriesOnly())
+            {
+                roxieTargetNames.append(clusterName.str());
```

Comment:
Both RTTQueries and RTTBoth should be added to roxieTargetNames but only RTTQueries should continue without adding to the queue list.

Reply:
Changed

Reply:
I thought the same thing - and that isn't what this code does.  If the type is both it will not be added to roxieTargetNames (afaics).

Reply:
I thought I fixed. Will do.